docs/options.md

```diff
@@ -0,0 +1,361 @@
+| --P_lr | float | 0.0002 | initial learning rate for P networks |
+
+
+## re_cycle_gan_semantic_mask
+
+| Parameter | Type | Default | Description |
+| --- | --- | --- | --- |
+| --lambda_A | float | 10.0 | weight for cycle loss (A -> B -> A) |
+| --lambda_B | float | 10.0 | weight for cycle loss (B -> A -> B) |
+| --lambda_identity | float | 0.5 | use identity mapping. Setting lambda_identity other than 0 has an effect of scaling the weight of the identity mapping loss. For example, if the weight of the identity loss should be 10 times smaller than the weight of the reconstruction loss, please set lambda_identity = 0.1 |
+| --use_label_B | flag |  | if true domain B has labels too |
+| --rec_noise | float | 0.0 | whether to add noise to reconstruction |
+| --out_mask | flag |  | use loss out mask |
+| --lambda_out_mask | float | 10.0 | weight for loss out mask |
+| --loss_out_mask | str | L1 | loss mask |
+| --charbonnier_eps | float | 1e-06 | Charbonnier loss epsilon value |
+| --disc_in_mask | flag |  | use in-mask discriminator |
+| --train_f_s_B | flag |  | if true f_s will be trained not only on domain A but also on domain B |
+| --fs_light | flag |  | whether to use a light (unet) network for f_s |
+| --lr_f_s | float | 0.0002 | f_s learning rate |
+| --nb_attn | int | 10 | number of attention masks |
+| --nb_mask_input | int | 1 | number of attention masks which will be applied on the input image |
+| --lambda_sem | float | 1.0 | weight for semantic loss |
+| --madgrad | flag |  | if true madgrad optim will be used |
+| --adversarial_loss_p | flag |  | if True, also train the prediction model with an adversarial loss |
+| --nuplet_size | int | 3 | Number of frames loaded |
+| --netP | str | unet_128 | specify P architecture [resnet_9blocks \| resnet_6blocks \| resnet_attn \| unet_256 \| unet_128] |
+| --no_train_P_fake_images | flag |  | if True, P wont be trained over fake images projections |
+| --projection_threshold | float | 1.0 | threshold of the real images projection loss below with fake projection and fake reconstruction losses are applied |
+| --P_lr | float | 0.0002 | initial learning rate for P networks |
+
+
+## segmentation
+
+| Parameter | Type | Default | Description |
+| --- | --- | --- | --- |
+| --lambda_A | float | 10.0 | weight for cycle loss (A -> B -> A) |
+| --lambda_B | float | 10.0 | weight for cycle loss (B -> A -> B) |
+| --lambda_identity | float | 0.5 | use identity mapping. Setting lambda_identity other than 0 has an effect of scaling the weight of the identity mapping loss. For example, if the weight of the identity loss should be 10 times smaller than the weight of the reconstruction loss, please set lambda_identity = 0.1 |
+
+
+## template
+
+| Parameter | Type | Default | Description |
+| --- | --- | --- | --- |
+| --lambda_regression | float | 1.0 | weight for the regression loss |
+
```

Comment:
It may need and update with last added options :)

Reply:
This file is generated, so it's as simple as running a script